src/popup/router/components/UrlStatus.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <Blacklisted v-if="type === 'blacklisted'" class="url-status" />
+  <Verified v-else-if="type === 'verified'" class="url-status" />
+  <NotSupported v-else-if="type === 'not-supported'" class="url-status" />
```

Comment:
```suggestion
  <Component :is="Icon" class="url-status" />
```
Otherwise, this will get messy after adding a click handler.